# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/moz.build
# Commit: 3daeacd08336
# Full Hash: 3daeacd0833682a474d9bd1adc8817474a7602be
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 6: Move NotificationObserver functions to NotificationUtils r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227691
# ==============================================================================

diff -r a497ddb030f6 -r 3daeacd08336 dom/notification/moz.build
--- a/dom/notification/moz.build	Thu Nov 07 17:32:05 2024 +0000
+++ b/dom/notification/moz.build	Thu Nov 07 17:32:05 2024 +0000
@@ -24,6 +24,7 @@
     "IPCUtils.h",
     "NotificationChild.h",
     "NotificationParent.h",
+    "NotificationUtils.h",  # Exported for ContentParent for now
 ]
 
 UNIFIED_SOURCES += [
